lib/ansible/modules/system/iptables.py

```diff
@@ -399,8 +399,14 @@ def construct_rule(params):
         False)
     append_match(rule, params['comment'], 'comment')
     append_param(rule, params['comment'], '--comment', False)
-    append_match(rule, params['ctstate'], 'state')
-    append_csv(rule, params['ctstate'], '--state')
+    if params['ctstate']:
```

Comment:
I understand the difference, mostly my concern is that existing plays are relying on it translating into `-m state --state` even though its kind of incorrectly named and `--ctstate` makes more sense, but I would make it conditional to the loading of conntrack itself (which needs to be loaded in both but explicitly for the command line matching --ctstate)